.ci/Jenkinsfile

```
@@ -33,13 +38,15 @@ pipeline {
         dir("${BASE_DIR}"){
           setEnvVar('GO_VERSION', readFile(".go-version").trim())
         }
+        setEnvVar('IS_BRANCH_AVAILABLE', isBranchUnifiedReleaseAvailable(env.BRANCH_NAME))
```

Once the release manager supports the DRA for this project, then we can set to `true` this, to validate the DRA works as expected before merging this PR.

```suggestion
        setEnvVar('IS_BRANCH_AVAILABLE', 'true')
```